lib/ansible/plugins/cliconf/iosxr.py

```diff
@@ -191,6 +191,11 @@ def commit(self, comment=None, label=None, replace=None):
                 cmd_obj['command'] = 'commit label {0}'.format(label)
             else:
                 cmd_obj['command'] = 'commit show-error'
+            # In some cases even a normal commit, i.e., !replace,
+            # throws a prompt and we need to handle it before
+            # proceeding further
+            cmd_obj['prompt'] = '(C|c)onfirm'
+            cmd_obj['answer'] = 'y'
```

Comment:
This API is odd... it seems to be, if I encounter this prompt, give it the answer.  But it doesn't require the prompt to occur.  Since that's what is desired here, there's no bug and I'll merge it.... Just commenting because that's not the standard way to write an API and it took me a bit to analyze the code to be sure that it was actually doing this.